### PR TITLE
[3.1] Mark components assemblies as non shipping

### PR DESCRIPTION
**Pull request by HaoK**
Follow up to https://github.com/dotnet/aspnetcore/pull/28492

Also mark WebAssembly packages as non shipping

### CODE REVIEW

File: src/Components/WebAssembly/JSInterop/src/Microsoft.JSInterop.WebAssembly.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <PackageTags>wasm;javascript;interop</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
+    <IsShipping>false</IsShipping>
```
Comment (HaoK) [inline src/Components/WebAssembly/JSInterop/src/Microsoft.JSInterop.WebAssembly.csproj:9]: @dougbu is this the same as IsShippingPackage ?

**Review by pranavkm — APPROVED** [review]

**Comment by pranavkm** [comment]
@wtgodbe / @dougbu can we merge infrastructure stuff to 3.1?

**Comment by wtgodbe** [comment]
> can we merge infrastructure stuff to 3.1?

No, branches are closed down for May. We should hold this until next month's check-in window

File: src/Components/WebAssembly/JSInterop/src/Microsoft.JSInterop.WebAssembly.csproj (hunk 1 above)
Comment (dougbu) [reply src/Components/WebAssembly/JSInterop/src/Microsoft.JSInterop.WebAssembly.csproj:9]: Sort of. The Arcade turns one into the other at https://github.com/dotnet/arcade/blob/e7ede87875f41a9b3df898ae08da5ebc96e24f56/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L22

**Comment by dougbu** [comment]
nit: @HaoK could you please use the GitHub UI features to make a bunch of suggestions in a comment, then batch committing those suggestions together❔ That'll take us from 15 emails to 2 when we're on your PRs 😀

**Pull request merged**